site/content/docs/v0.0.7/install.md

```diff
@@ -0,0 +1,495 @@
+# Installing Cartographer
+
+There are three recommended ways of installing Cartographer:
+
+1. with a single YAML file (quick!)
+2. using an [imgpkg bundle], making easier for relocation
+3. using a [carvel Packaging] objects, for an opinionated package-based
+   workflow
+
+If all you want to do is get started quick and fetching images from DockerHub
+is not an issue, [`1`](#1-single-file-quick-install) is the way to go.
+Otherwise, check out the other approaches (especially if you want to have
+everything relocated to a registry of your own).
+
+tl;dr _(having [cert-manager] already installed)_:
+
+```bash
+kubectl create namespace cartographer-system
+kubectl apply -f https://github.com/vmware-tanzu/cartographer/releases/latest/download/cartographer.yaml
```

Comment:
```suggestion
kubectl apply -f https://github.com/vmware-tanzu/cartographer/releases/download/v0.0.7/cartographer.yaml
```

Reply:
Yeah theres a cool script I eventually want to repurpose over on velero: https://github.com/vmware-tanzu/velero/blob/main/hack/release-tools/gen-docs.sh